community/blog/2022-06 .NET 6 finally comes to Arcus Templates in v0.8 release/README.md

```diff
@@ -0,0 +1,112 @@
+# .NET 6 finally comes to Arcus Templates in v0.8 release, together with new templates and features
+The v0.8 release of Arcus Templates ends the journey of adding .NET support to Arcus. This release also contains tons of new features and useful changes. This run-down post will guide you through the biggest ones.
+
+## New Azure Functions Service Bus message handling project templates
+The core functionality of the Arcus Messaging library; is message handling. Describing the business logic in reusable message handlers where an incoming message is routed via Arcus functionality to the correct handler. Previous releases of Arcus Messaging made sure that the functionality that receives a message is separated from the functionality that routes the message. The reason for this is so we can reuse the message routing in different scenarios. This is one of those scenarios.
+
+Azure Functions differ from the usual message pump setup because the message is already being received by the trigger function; whether it is a message on an Azure Service Bus queue or a topic subscription. Since the trigger function already takes over the work of the message pump, we only need to route messages.
+
+Easy extensions on the Azure Functions types allow you to register your message handlers the same way as with a message pump setup. 
```

Comment:
```suggestion
Easy extensions on the `IFunctionsHostBuilder` allow you to register your message handlers the same way as with a message pump setup. 
```

Reply:
Yes, was thinking to include this, although, it is not the only type that is being extended (or would be extended), so not sure to mention it?

Reply:
But it is maybe the main type ? Maybe this then ?

Reply:
Yeah, currently it's still only this type, so I applied your suggestion. 😉 .